applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_pw_line_element.cpp

```diff
@@ -71,7 +75,8 @@ intrusive_ptr<Element> CreatePwLineElementWithoutPWDofs(ModelPart& rModelPart, c
     nodes.push_back(rModelPart.CreateNewNode(1, 1.0, 1.0, 0.0));
     const auto                                 p_geometry = std::make_shared<Line2D2<Node>>(nodes);
     const std::vector<CalculationContribution> contirbutions = {
```

Comment:
There is a typo here (it was already there):
```suggestion
    const std::vector<CalculationContribution> contributions = {
```

Reply:
Done